### PR TITLE
GEOS-3.7.2 SWIG-4.0.0 foss-2019a

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/SWIG/SWIG-4.0.0-foss-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name = 'SWIG'
+version = '4.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```
Comment (akesandgren) [inline easybuild/easyconfigs/s/SWIG/SWIG-4.0.0-foss-2019a-Python-3.7.2.eb]: SWIG should be att GCCcore level, see https://github.com/easybuilders/easybuild-easyconfigs/pull/7923

File: easybuild/easyconfigs/s/SWIG/SWIG-4.0.0-foss-2019a-Python-3.7.2.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name = 'SWIG'
+version = '4.0.0'
```
Comment (Micket) [inline easybuild/easyconfigs/s/SWIG/SWIG-4.0.0-foss-2019a-Python-3.7.2.eb]: There is a SWIG 3.0.12 at GCCcore/8.2.0

If we need a this newer version (which I suppose is OK, as it's just build-dep), then it should be at GCCcore level from now.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-foss-2019a-Python-3.7.2.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'GEOS'
+version = '3.7.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://trac.osgeo.org/geos'
+description = """GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```
Comment (Micket) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-foss-2019a-Python-3.7.2.eb]: I don't think there is any MPI or BLAS stuff in this package. Perhaps we could move it down to GCC level?

File: easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-foss-2019a-Python-3.7.2.eb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+source_urls = ['http://download.osgeo.org/geos/']
+sources = [SOURCELOWER_TAR_BZ2]
+patches = ['GEOS-%(version)s_fix-Python3.patch']
+checksums = [
+    '2166e65be6d612317115bfec07827c11b403c3f303e0a7420a2106bc999d7707',  # geos-3.7.2.tar.bz2
+    'e14b54796d9d41261caae64b5a106b4bd8a77f37a51aa9b8ada30d87d208e2e0',  # GEOS-3.7.2_fix-Python3.patch
+]
+
+dependencies = [('Python', '3.7.2')]
```
Comment (Micket) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-foss-2019a-Python-3.7.2.eb]: I was actually working on a multi_deps version (but stopped when the GDAL/PROJ/... packages turned into a mess, which it why I suggested we would remove rgdal from R).

```
builddependencies = [
    ('binutils', '2.31.1'),
    ('SWIG', '3.0.12', '-Python-%(pyver)s'),
]
multi_deps = {'Python': ['3.7.2', '2.7.15']}
```
but it required some fix to framework/block. I think these might have been fixed in develop branch already.

@boegel Thoughts on this?

Comment (boegel) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-foss-2019a-Python-3.7.2.eb]: @Micket The problem you're referring to should be fixed with https://github.com/easybuilders/easybuild-framework/pull/2854

Having a single `GEOS` easyconfig using `*/2019a` for both Python 2.x and 3.x would be nicer, indeed.

Comment (fizwit) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-foss-2019a-Python-3.7.2.eb]: rgal in R is holding up the 3.6.0 foss/2019a release for sure. I will close this for now and rebuild GEOS with SWIG/3.0.12 8.2.0

**Pull request closed without merging**